lfx-mentorship/2022/03-Sept-Nov/README.md

```diff
@@ -117,9 +117,9 @@ Description: Network topologies and graph databases go hand-in-hand. The OpenAPI
 - a Kubernetes ontology using OWL as a popular (and mature) way of doing this.
 - a cuelang-based component generator
 
-Recommended Skills: cuelang, golang, neo4j
-Mentor(s): [Lee Calcote](https://github.com/leecalcote), [Matt Young](https://github.com/halcyondude)
-Issue: https://github.com/cncf/tag-network/issues/21
+- Recommended Skills: cuelang, golang, neo4j
```

Comment:
We should add a `- ` to line 115 (the description), and then indent lines 117 1nd 118 as well.